java/io/bazel/rules/closure/testing/TestDriver.java

```diff
@@ -53,23 +52,12 @@ public boolean run() {
           .pollingEvery(Duration.ofMillis(POLL_INTERVAL))
           .withTimeout(Duration.ofSeconds(TEST_TIMEOUT))
           .until(
-              executor -> {
-                testFinishedSuccessfully =
-                    (boolean) executor.executeScript("return window.top.G_testRunner.isFinished()");
-                if (!testFinishedSuccessfully) {
-                  logger.log(Level.SEVERE, "G_testRunner has not finished successfully");
-                }
-                return true;
-              });
+              executor -> (boolean) executor.executeScript("return window.top.G_testRunner.isFinished()")
+          );
     } catch (TimeoutException e) {
-      testFinishedSuccessfully = false;
       logger.log(Level.SEVERE, String.format("Test timeout after %s seconds", TEST_TIMEOUT));
```

Comment:
We also need to return false here. There is no much point of trying to dig the results if it has timed out